tensorflow/python/ops/nn_ops.py

```diff
@@ -1294,7 +1294,7 @@ def conv3d_transpose(value,
     padding: A string, either `'VALID'` or `'SAME'`. The padding algorithm.
       See the @{tf.nn.convolution$comment here}
     data_format: A string, either `'NDHWC'` or `'NCDHW`' specifying the layout
-      of the input and output tensors. Defaults to `'NDHWC'`.
```

Comment:
`max_pool`, `avg_pool`, `conv2d_tranpose` and many other ops have `data_format="NHWC"` specified in function signature. I've done the same for this for the sake of consistency.

/ping: @martinwicke :rocket: